### ISSUE TITLE
hoot new error

### ISSUE THREAD

**Issue by Matt-Gleich** [rishiosaur/hoot#23]
When running the command `hoot new` I get the following stack trace:

```
(node:55837) UnhandledPromiseRejectionWarning: TypeError: item.toLowerCase is not a function
    at validateItem (/usr/local/lib/node_modules/hoot-cli/bin/commands/utility/newItem.js:8:30)
    at Command.newItem (/usr/local/lib/node_modules/hoot-cli/bin/commands/utility/newItem.js:32:5)
    at Command.listener (/usr/local/lib/node_modules/hoot-cli/node_modules/commander/index.js:291:8)
    at Command.emit (events.js:315:20)
    at Command.parseArgs (/usr/local/lib/node_modules/hoot-cli/node_modules/commander/index.js:672:12)
    at Command.parse (/usr/local/lib/node_modules/hoot-cli/node_modules/commander/index.js:459:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/hoot-cli/bin/index.js:53:9)
    at Module._compile (internal/modules/cjs/loader.js:1176:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1196:10)
    at Module.load (internal/modules/cjs/loader.js:1040:32)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:55837) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:55837) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by rishiosaur** [comment]
We can add a hook in `index.js` to help resolve this with a custom error message.

**Comment by LoboAdrian** [comment]
comment

**Comment by rishiosaur** [comment]
I've fixed this with ac2e825. Closing the issue.
